model/metrics/http.yaml

```diff
@@ -6,36 +6,18 @@ groups:
     attributes:
       - ref: server.address
         requirement_level: opt_in
-        note: |
-          Determined by using the first of the following that applies:
-
-          - The [primary server name](/docs/http/http-spans.md#http-server-definitions) of the matched virtual host.
-          - Host identifier of the [request target](https://www.rfc-editor.org/rfc/rfc9110.html#target.resource)
-            if it's sent in absolute-form.
-          - Host identifier of [Forwarded#host](https://developer.mozilla.org/docs/Web/HTTP/Headers/Forwarded#host),
-            [X-Forwarded-Host](https://developer.mozilla.org/docs/Web/HTTP/Headers/X-Forwarded-Host), or a similar header.
-          - Host identifier of the `Host` header.
-
-          MUST NOT include the port identifier.
+        note: >
+          See [Setting `server.address` and `server.port` attributes](/docs/http/http-spans.md#setting-serveraddress-and-serverport-attributes).
 
-          Warning: since this attribute may be based on HTTP headers, opting in to it may allow an attacker
+          > **Warning**: since this attribute may be based on HTTP headers, opting in to it may allow an attacker
```

Comment:
If you want the Warning here to contain the nice icon, this has to be on its own like, like

> **Warning**
> since this attribute..

Reply:
cool, thanks for the pointer!